pkg/utils/node/predicates.go

```diff
@@ -31,7 +31,28 @@ func IsReadyAndSchedulable(node v1.Node) bool {
 	return false
 }
 
-func IsPastEmptyTTL(node *v1.Node) bool {
+func IsReady(node v1.Node) bool {
+	for _, condition := range node.Status.Conditions {
+		if condition.Type == v1.NodeReady {
+			return condition.Status == v1.ConditionTrue
+		}
+	}
+	return false
+}
+
+func FailsToBecomeReady(node v1.Node) bool {
+	if time.Since(node.GetCreationTimestamp().Time) < (15 * time.Minute) {
```

Comment:
Can we pass this 15 min duration thru from maybe the reallocation controller (or if you don't want to pass thru that deep, at least in utilization.go)? The comment in the reallocation controller says 15 minutes but you have to dig down 2 more levels to actually see how to change the duration.  So I think the comment could get stale at some point or it'll just be awkward to configure this value.

Reply:
You make a good point. While I don't think we want to make this super configurable, I would not like to lose the ability in levels, so I'll make this a parameter to pass in. Thanks!